tms_django/apps/tms/tests/test_models.py

```diff
@@ -0,0 +1,49 @@
+from pprint import pprint
+
+from apps.tms.models import (
+    BookedLoad,
+    Broker,
+    DispatcherProfile,
+    DriverProfile,
+    Truck,
+    TruckKind,
+)
+from apps.tms.tests.factories import (
+    BookedLoadFactory,
+    BrokerFactory,
+    DispatcherProfileFactory,
+    DriverProfileFactory,
+    TruckFactory,
+)
+from django.test import TestCase
+
+
```

Comment:
@elya-hp please add tests for custom id fields where you applied generators, such as `load_id` `generate_load_id` and similar

Reply:
done